app/main/views/users.py

```diff
@@ -87,42 +93,74 @@ def download_users(framework_slug):
 @login_required
 @role_required('admin')
 def download_buyers_and_briefs():
-    buyers = (user for user in data_api_client.find_users_iter(role='buyer'))
-    briefs = (brief for brief in data_api_client.find_briefs_iter(with_users=True))
-    buyer_headings = [
+    users = {user["id"]: dict(user, briefs=[]) for user in data_api_client.find_users_iter(role="buyer")}
+
+    # join users with briefs (a "hash join")
+    for brief in data_api_client.find_briefs_iter(with_users=True):
+        for user in brief["users"]:
+            users[user["id"]]["briefs"].append(brief)
+
+    timestamp = datetime.utcnow().strftime('%Y%m%dT%H%M%S')
```

Comment:
This is inherited from previous code - I looked at whether we could do it less arbitrarily and I think I concluded we couldn't. It's not using `DATETIME_FORMAT` because it needs to be used for a filename and therefore restricts its "funny characters". Should possibly add a comment about this.